tests/unit/model/test_yolo.py

```diff
@@ -0,0 +1,190 @@
+from numpy.testing import assert_array_almost_equal
+
+from datachain.model.yolo import (
+    YoloBox,
+    YoloCls,
+    YoloObb,
+    YoloPose,
+    YoloPoseBodyPart,
+    YoloSeg,
+)
+
+
+def test_yolo_box():
+    model = YoloBox(
+        cls=[1, 2],
+        name=["person", "chair"],
+        confidence=[0.8, 0.5],
+        box=[
+            [10, 20, 190, 80],
+            [160, 80, 200, 100],
+        ],
+        orig_shape=[100, 200],
+    )
+
+    assert model.model_dump() == {
+        "cls": [1, 2],
+        "name": ["person", "chair"],
+        "confidence": [0.8, 0.5],
+        "box": [
+            [10, 20, 190, 80],
+            [160, 80, 200, 100],
+        ],
+        "orig_shape": [100, 200],
+    }
+
+    assert model.img_size == (200, 100)
+
+    alb_boxes = list(model.to_albumentations())
+    assert len(alb_boxes) == 2
+    assert_array_almost_equal(alb_boxes[0], [0.05, 0.2, 0.95, 0.8], decimal=3)
+    assert_array_almost_equal(alb_boxes[1], [0.8, 0.8, 1.0, 1.0], decimal=3)
+
+    coco_boxes = list(model.to_coco())
+    assert len(coco_boxes) == 2
+    assert coco_boxes[0] == [10, 20, 180, 60]
+    assert coco_boxes[1] == [160, 80, 40, 20]
+
+    voc_boxes = list(model.to_voc())
+    assert len(voc_boxes) == 2
+    assert voc_boxes[0] == [10, 20, 190, 80]
+    assert voc_boxes[1] == [160, 80, 200, 100]
+
+    yolo_boxes = list(model.to_yolo())
+    assert len(yolo_boxes) == 2
+    assert_array_almost_equal(yolo_boxes[0], [0.5, 0.5, 0.9, 0.6], decimal=3)
+    assert_array_almost_equal(yolo_boxes[1], [0.9, 0.9, 0.2, 0.2], decimal=3)
+
+
+def test_yolo_obb():
```

Comment:
Q: what to we test with this kind of tests?

Reply:
We are testing if models were not changed, since other code may rely on them.